lib/job.js

```diff
@@ -241,20 +242,26 @@ class Job extends BaseModel {
                 count: MAX_COUNT
             }
         };
-        const findMetrics = ({ id, jobId, eventId, createTime, status, startTime, endTime }) => {
+        const findMetrics = (build, aggregate) => {
```

Comment:
can this function be moved outside of this function ?

Reply:
Can do. 
But what's the reason? The function is not too long. Will need to pass in additional param (stepName) if move it outside :(

Reply:
I generally avoid nested functions. Readability wise great, but not so for performance since these function objects are created on each run.

Reply:
Okay 👍 
I refactored